controller_manager:
  ros__parameters:
    update_rate: 250  # Hz
    use_sim_time: true

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

    position_controller:
      type: position_controllers/JointGroupPositionController

position_controller:
  ros__parameters:
    joints: [base_to_rotary, rotary_to_lower, lower_to_upper, upper_to_support, support_to_wrist ]
    command_interfaces: [position]
    state_interfaces: [position, velocity]